dotcom-rendering/src/components/Discussion/Comments.tsx

```diff
@@ -33,6 +36,7 @@ type Props = {
 	onRecommend?: (commentId: number) => Promise<boolean>;
 	onPreview?: typeof preview;
 	idApiUrl: string;
+	renderingTarget: RenderingTarget;
```

Comment:
You do not need to pass the `renderingTarget` as it is available inside every Island from `useConfig`

Reply:
We tried that initially, but got this error: `useConfig must be used within the ConfigContext provider`

Reply:
This is [quite suprising](https://github.com/guardian/dotcom-rendering/blob/db309c21ac3965c0427c303cba447e1a8bb867e9/dotcom-rendering/src/client/islands/getConfig.ts#L3-L28)… was that error thrown when running `make dev`?

Reply:
It was a testing issue 🤦‍♀️ I've sorted it now